src/mqttClient.ts

```diff
@@ -98,6 +99,9 @@ export function initializeMQTT(eventEmitter: EventEmitter) {
           trashbin[0].fillLevel = fillLevel ? Math.round(fillLevel * 100) : 0;
           await trashbin[0].save();
         }
+        await axios.put('http://localhost:5001/api/v1/trashbin/updateFillLevelChanges', {
```

Comment:
It is definitely an interesting idea to use axios to call the url of the backend.
Latency wise this is not an optimal solution and it also creates unnecessary overhead.
Please have a look at the `src/service` folder. For example there is already a function called "generateUniqueTrashbinIdentifier" implemented there. This function can then be imported and used in other places of the backend.
Please refactor your code, to also use this service structure to update fillLevelChanges and then import the defined function in the mqtt and trashbin controller file (and dont forget to remove axios from the package.json).
Perhaps you could even make it so, that when a new measurement comes in, not all fillLevelChanges are updated but just the one of the corresponding trashbin (atleast I think, that currently all levelChanges are updated, but perhaps Im misunderstanding).